src/main/git-elegant-check

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+default() {
+    if ["$1"=="-a"] || ["$1"=="-all"]; then
```

Comment:
@ivan-podorozhnyi Please, add whitespaces in `test` command. It won't work this way.
Also make sure, you are not messed with scores in commands. Seems word-commands have to have two scores preceding.